src/Tribe/Cache.php

```diff
@@ -165,8 +167,7 @@ public function delete_expired_transients() {
 		global $wpdb;
 
 		$time = time();
-
-		$sql = "
+		$sql  = "
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 07cc09fcdf9df8ea1bca9bc575792a1e</sub>